src/main/resources/db/migration/V42_1__Create_dummy_table.sql

```diff
@@ -6,4 +6,4 @@ create table if not exists dummy
 
 insert into dummy (id)
 values ('dummy-table-id-1')
-on conflict on constraint dummy_pk do nothing;
```

Comment:
why did you do that ?
it will raise conflicts with existing history

put a breaking change if it is really necessary